pkgs/desktops/gnome/extensions/argos/default.nix

```diff
@@ -1,14 +1,19 @@
-{ fetchFromGitHub, lib, stdenv }:
+{ fetchFromGitHub, lib, stdenv, unstableGitUpdater }:
+
+# There are no tagged releases anymore so we're using unstable
+# but having this keep getting updated for every upstream commit
+# seems pointless.
```

Comment:
It doesn't update for every commit and has a delay of IIRC a week or so

Reply:
Removed this part from both argos and pop-shell packages 👍🏼